devicetypes/smartthings/nyce-open-closed-sensor.src/nyce-open-closed-sensor.groovy

```diff
@@ -253,34 +241,6 @@ private List parseIasMessage(String description) {
 	log.debug "parseIasMessage: Trouble Status ${zs.trouble}"
 	log.debug "parseIasMessage: Sensor Status ${zs.alarm1}"
 
-	/* 	Comment out this path to check the battery state to avoid overwriting the
-		battery value (Change log #2), but keep these conditions for later use
-	 resultMap_battery_state.name = "battery_state"
-	 if (zs.isTroubleSet()) {
-		 resultMap_battery_state.value = "failed"
-
-		 resultMap_battery.name = "battery"
-		 resultMap_battery.value = 0
-	 }
-	 else {
-		 if (zs.isBatterySet()) {
-			 resultMap_battery_state.value = "low"
-
-			 // to generate low battery notification by the platform
-			 resultMap_battery.name = "battery"
-			 resultMap_battery.value = 15
-		 }
-		 else {
-			 resultMap_battery_state.value = "ok"
-
-			 // to clear the low battery state stored in the platform
-			 // otherwise, there is no notification sent again
-			 resultMap_battery.name = "battery"
-			 resultMap_battery.value = 80
-		 }
-	 }
-	*/
-
 	resultListMap << resultMap_battery_state
```

Comment:
This function can be cleaned up further since only `resultMap_sensor` is set

Reply:
ok, removed. But I left log.debugs.